pkg/apis/authorization/types.go

```diff
@@ -38,6 +38,10 @@ type SubjectAccessReview struct {
 	Status SubjectAccessReviewStatus
 }
 
+// +genclient=true
+// +nonNamespaced=true
+// +noMethods=true
```

Comment:
> Sorry, I couldn't find documentation on what +noMethods accomplishes. Does it limit the actions generated to "Create"?

It prevents the client generator from generating methods.  Since this isn't a normal resource, it doesn't have types for things like `SelfSubjectAccessReviewList` (no get method), so the default generator fails.

It means you have to manually create the `Create` methods in the generated_expansion.go file.